test/index-test.js

```diff
@@ -4,10 +4,12 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 import jsdom from 'mocha-jsdom'
 import FontAwesome from '../src'
+import styles from '../src/react-fontawesome.css'
```

Comment:
This is required to check local CSS class name in https://github.com/danawoodman/react-fontawesome/pull/30/files#diff-1c5726b71a63db52a608b0c2c9726644R160.